src/transaction.ts

```diff
@@ -198,6 +201,9 @@ export class Transaction implements TransactionStorageStructure {
   nonParticipation?: boolean;
   group?: Buffer;
   extraPages?: number;
+  stateProofType?: number | bigint;
```

Comment:
why not use just `number`?

Reply:
We use the `number | bigint` union type for `assetTotal` and `amount` which are also `uint64` underlying types. Seems fine to use it here as well.